$NetBSD: patch-as,v 1.3 2008/10/10 21:58:43 he Exp $

asm/page.h is no longer installed as part of the kernel header
page on Gentoo/Linux.

--- ext/IPC/SysV/SysV.xs.orig	2007-12-18 11:47:07.000000000 +0100
+++ ext/IPC/SysV/SysV.xs
@@ -24,8 +24,6 @@
 #      elif defined(HAS_GETPAGESIZE)
 #          undef  SHMLBA /* not static: determined at boot time */
 #          define SHMLBA getpagesize()
-#      elif defined(__linux__)
-#          include <asm/page.h>          
 #      endif
 #   endif
 #endif
